activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -21,7 +21,29 @@ module ActiveSupport
   #   key   = ActiveSupport::KeyGenerator.new('password').generate_key(salt, len) # => "\x89\xE0\x156\xAC..."
   #   crypt = ActiveSupport::MessageEncryptor.new(key)                            # => #<ActiveSupport::MessageEncryptor ...>
   #   encrypted_data = crypt.encrypt_and_sign('my secret data')                   # => "NlFBTTMwOUV5UlA1QlNEN2xkY2d6eThYWWh..."
-  #   crypt.decrypt_and_verify(encrypted_data)                                    # => "my secret data"
+  #   crypt.decrypt_and_verify(encrypted_data) `                                  # => "my secret data"
+  #
+  # MessageEncryptor supports the following metadata that's associated with a message:
+  #
+  # * <tt>:purpose</tt> - The purpose for which the message is intended to be used.
+  # * <tt>:expires_at</tt> - The expiration date of the message.
+  # * <tt>:expires_in</tt> - The relative expiration date. Note that, :expires_at takes
```

Comment:
Maybe you can surround `:expires_at` and `:expires_in` with a `+` sign to make them formatted on the generated documentation.